src/GafferOSL/OSLShader.cpp

```diff
@@ -455,7 +455,27 @@ Plug *loadCompoundNumericParameter( const OSLQuery::Parameter *parameter, const
 		return existingPlug;
 	}
 
-	typename PlugType::Ptr plug = new PlugType( name, parent->direction(), defaultValue, minValue, maxValue, Plug::Default | Plug::Dynamic );
+	typename PlugType::Ptr plug;
```

Comment:
I guess this is what you would want to move over to your `OpenImageIOAlgo.h`?

https://github.com/GafferHQ/gaffer/blob/master/src/GafferOSL/ShadingEngine.cpp#L98